ResearchKit/Common/ORKDefines_Private.h

```diff
@@ -51,4 +51,7 @@ ORK_EXTERN NSDateComponents *ORKTimeOfDayComponentsFromString(NSString *string)
 
 ORK_EXTERN NSDateFormatter *ORKResultDateTimeFormatter() ORK_AVAILABLE_DECL;
 ORK_EXTERN NSDateFormatter *ORKResultTimeFormatter() ORK_AVAILABLE_DECL;
-ORK_EXTERN NSDateFormatter *ORKResultDateFormatter() ORK_AVAILABLE_DECL;
\ No newline at end of file
+ORK_EXTERN NSDateFormatter *ORKResultDateFormatter() ORK_AVAILABLE_DECL;
+
+#define _ORK_HTMLfromTemplate(name) \
+NSString stringWithFormat:[ORKBundle() localizedStringForKey:(name) value:@"" table:@"HTMLTemplates"]
```

Comment:
That's what I meant, private method that takes a va_list and passes it to NSBundle stringWithFormat